### PR TITLE
Changes needed for RFC 48

### PR DESCRIPTION
**Pull request by shaobo-he-aws**
*Issue #, if available:*

*Description of changes:*

### CODE REVIEW

File: cedar-policy-generators/src/schema.rs [hunk 1]
```diff
@@ -152,6 +153,7 @@ fn arbitrary_typeofattribute_with_bounded_depth<N: From<ast::Name>>(
     Ok(json_schema::TypeOfAttribute {
         ty: arbitrary_schematype_with_bounded_depth::<N>(settings, entity_types, max_depth, u)?,
         required: u.arbitrary()?,
+        annotations: Annotations::new(),
```
Comment (john-h-kastner-aws) [inline cedar-policy-generators/src/schema.rs:156]: could generate some arbitrary annotations here for the schema roundtrip targets

Comment (shaobo-he-aws) [reply cedar-policy-generators/src/schema.rs:156]: Yeah. I will create an issue and set up another PR for it. This PR is to ensure RFC 48 doesn't cause issues for existing properties.

**Review by john-h-kastner-aws — APPROVED** [review]

File: cedar-policy-generators/src/schema.rs (hunk 1 above)
Comment (shaobo-he-aws) [reply cedar-policy-generators/src/schema.rs:156]: Issue: https://github.com/cedar-policy/cedar-spec/issues/493

File: cedar-drt/fuzz/src/schemas.rs [hunk 2]
```diff
@@ -189,6 +189,12 @@ impl<K: Eq + Ord + Display, V: Equiv> Equiv for BTreeMap<K, V> {
     }
 }
 
+impl<N: Clone + PartialEq + Debug + Display + TypeName + Ord> Equiv for json_schema::CommonType<N> {
+    fn equiv(lhs: &Self, rhs: &Self) -> Result<(), String> {
+        Equiv::equiv(&lhs.ty, &rhs.ty)
+    }
```
Comment (cdisselkoen) [inline cedar-drt/fuzz/src/schemas.rs:195]: should we check more than just the `.ty` fields?  is the other field annotations, should we check that the annotations are the same?

**Review by cdisselkoen — APPROVED** [review]

File: cedar-drt/fuzz/src/schemas.rs (hunk 2 above)
Comment (shaobo-he-aws) [reply cedar-drt/fuzz/src/schemas.rs:195]: We'll do it later: https://github.com/cedar-policy/cedar-spec/pull/492#discussion_r1887196692

**Pull request merged**